.github/workflows/test-lang-c++.yml

```diff
@@ -0,0 +1,44 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Minor thing, but I would suggest to name cpp.yml? Having +'es in the filename feels a bit weird to me.

Reply:
I can. Do you want me to move the lang directory as well?

Reply:
@Fokko Happy New Year! I'm going to leave this as-is, for consistency with the lang/c++ directory, which I think would be out of scope to change right now. If you want to rename the lang to cpp, let's discuss that in another thread on dev@